dot/sync/tip_syncer_integeration_test.go

```diff
@@ -20,8 +21,8 @@ import (
 )
 
 func newTestTipSyncer(t *testing.T) *tipSyncer {
-	finHeader, err := types.NewHeader(common.NewHash([]byte{0}),
-		trie.EmptyHash, trie.EmptyHash, 200, types.NewDigest())
+	finHeader, err := types.NewHeader(common.NewHash([]byte{0}), trie.EmptyHash, trie.EmptyHash, 200,
+		types.NewDigest())
```

Comment:
supernit unneeded diffs

```suggestion
	finHeader, err := types.NewHeader(common.NewHash([]byte{0}),
		trie.EmptyHash, trie.EmptyHash, 200, types.NewDigest())
```

Reply:
Reverted.